app/components/questions/RentBurdenRates.js

```diff
@@ -199,6 +229,7 @@ export default function RentBurdenedRates(props) {
       <div className="center medium-width">
         <h2>Rent Burdened Households Rate by Area</h2>
         <AreaSelector data={data} handleChange={setSelectedArea} />
+        <YearSelector data={years} handleChange={setSelectedYear} />
```

Comment:
Do these selectors show up on the same or different lines? Could use some styling. Would be great to see a screenshot with the PR to know how it looks.

Reply:
![temp](https://user-images.githubusercontent.com/84047863/127438114-98040eae-a54f-4bca-81a1-e397281e6e20.PNG)
I think I will change it a bit.